client/blocks/daily-post-button/test/index.jsx

```diff
@@ -61,13 +71,15 @@ describe( 'DailyPostButton', () => {
 					markPostSeen={ markPostSeen }
 				/>
 			);
-			expect( renderAsSpan.type() ).toEqual( 'span' );
+
+			const article = screen.queryByRole( 'article' );
+			expect( container.firstChild ).toBe( article );
```

Comment:
Just like above for the `container.firstChild.tagName`

Reply:
Fixed via https://github.com/Automattic/wp-calypso/pull/64607/commits/c4aa9e67e21bcf7f4bfc59dcc745ee3e7cd6fed6